sdk/include/aditof/frame.h

```diff
@@ -136,6 +136,15 @@ class Frame {
     SDK_API virtual Status getAttribute(const std::string &attribute,
                                         std::string &value) const;
 
+    /**
+     * @brief Gets the temperature values stored in the metadata for each frame
```

Comment:
A slightly better rewording would be: "Extracts the sensor and laser temperatures from metadata within this frame."

Reply:
Fixed.